Library/Formula/b2-tools.rb

```diff
@@ -13,12 +13,14 @@ class B2Tools < Formula
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec+"lib/python2.7/site-packages"
+
+    system "python", "setup.py", "install", "--prefix=#{libexec}"
```

Comment:
Bit weird. The return value of `Language::Python.setup_install_args(libexec)` is following:

``` ruby
["-c", "import setuptools, tokenize\n__file__ = 'setup.py'\nexec(compile(getattr(tokenize, 'open', open)(__file__).read()\n  .replace('\\r\\n', '\\n'), __file__, 'exec'))\n", "--no-user-cfg", "install", "--prefix=/usr/local/Cellar/b2-tools/0.4.4/libexec", "--single-version-externally-managed", "--record=installed.txt"]
```